1209/F00D/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Shifter/Pedals Adapter 
+owner: RomanStepanov
```

Comment:
Please set this to the directory name of the org file - in this case, 'robotsrulz'.